### PR TITLE
localization for the skipped screen

### PR DESCRIPTION
**Pull request by captainamiedi**
Fixes #
Localization for review screen, select species screen and other errors

Changes in this pull request:
- Review component
- Select species component
- Tree inventory component

@

### CODE REVIEW

File: app/languages/en/index.js [hunk 1]
```diff
@@ -9,6 +9,8 @@ import DownloadMap from './user/downloadMap.json';
 import ManageUser from './user/manageUser.json';
 import SavedAreas from './user/savedAreas.json';
 import Permission from './static/permission.json';
+import Review from './user/treeReview.json';
```
Comment (norbertschuler) [inline app/languages/en/index.js:12]: I think this needs to be changed in every of these similar files in every language folder.

File: android/gradle/wrapper/gradle-wrapper.properties [hunk 2]
```diff
@@ -1,5 +1,5 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https://services.gradle.org/distributions/gradle-6.2-all.zip
+distributionUrl=https://services.gradle.org/distributions/gradle-5.6.4-all.zip
```
Comment (norbertschuler) [inline android/gradle/wrapper/gradle-wrapper.properties:3]: Why did you downgrade?

Comment (captainamiedi) [reply android/gradle/wrapper/gradle-wrapper.properties]: i have made the necessary changes

File: app/languages/en/index.js (hunk 1 above)
Comment (captainamiedi) [reply app/languages/en/index.js:12]: Yes. it needs to be translated

Comment (norbertschuler) [reply app/languages/en/index.js:12]: I meant there is an index.js file in every language folder which needs to be changed like this file.

**Comment by norbertschuler** [comment]
I changed LingoHub to pull new resources from the develop branch now, so we need to merge this in order to do the translations.

**Pull request merged**